runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -659,12 +613,197 @@ public synchronized CorfuRuntime connect() {
 
         checkVersion();
 
-        if (loadSmrMapsAtConnect) {
+        if (parameters.isUseFastLoader()) {
             FastObjectLoader fastLoader = new FastObjectLoader(this)
-                    .setBatchReadSize(getBulkReadSize())
-                    .setTimeoutInMinutesForLoading(timeoutInMinutesForFastLoading);
+                    .setBatchReadSize(parameters.getBulkReadSize())
+                    .setTimeoutInMinutesForLoading((int) parameters.fastLoaderTimeout.toMinutes());
             fastLoader.loadMaps();
         }
         return this;
     }
+
+    // Below are deprecated methods which should no longer used and may be deprecated in the future.
+
+    // region Deprecated Constructors
+    /**
+     * Constructor for CorfuRuntime.
+     *
+     **/
+    @Deprecated
+    public CorfuRuntime() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)